core/domain/prod_validation_jobs_one_off.py

```diff
@@ -3024,9 +2891,7 @@ def _get_model_id_regex(cls, item):
 
     @classmethod
     def _get_change_domain_class(cls, item):
```

Comment:
Do you even need this to be a separate method, now that there's just one "thing"?

Reply:
I'll remove the `if` condition and return just `SkillChange`. I did see other places where the `_get_change_domain_class` just returned a single change object.

Reply:
OK, thanks for checking!